OpenRA.Mods.Common/Effects/Bullet.cs

```diff
@@ -31,8 +31,8 @@ public class BulletInfo : IProjectileInfo
 		[Desc("Image to display.")]
 		public readonly string Image = null;
 
-		[Desc("Loop this sequence of Image while this projectile is moving.")]
-		[SequenceReference("Image")] public readonly string Sequence = "idle";
+		[Desc("Loop these sequences of Image while this projectile is moving.")]
+		[SequenceReference("Image")] public readonly string[] Sequences = { "idle" };
```

Comment:
Whoops, indeed. I'll get my dunce cap. The upgrade rule does seem justified then.